docker/Dockerfile

```diff
@@ -177,6 +177,18 @@ RUN wget -O /tmp/open-vmdk-master.zip https://github.com/vmware/open-vmdk/archiv
     unzip -d /tmp/ /tmp/open-vmdk-master.zip && \
     cd /tmp/open-vmdk-master/ && make && make install
 
+# Packages need for build live-build
+RUN apt-get update && apt-get install -y \
+      cpio
+
+RUN wget https://github.com/vyos/vyos-live-build/commit/3ddf0e979c352f2917be6a949fb8dd37b5c9f2b7.patch \
```

Comment:
Until we decide whether we really need a full-blown fork of live-build, maybe just put the patch in this repository?

Now it's a bit of a "worst of both worlds" situation: it requires a full fork but is also pulling the upstream repo. If we are ready to pull the upstream repo, the fork is not useful.

Reply:
fixed, now the patch file is stored in the docker context dir